pkg/sdn/plugin/node_iptables.go

```diff
@@ -95,6 +95,7 @@ func (n *NodeIPTables) getStaticNodeIPTablesRules() []FirewallRule {
 		{"filter", "INPUT", []string{"-p", "udp", "-m", "multiport", "--dports", VXLAN_PORT, "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", TUN, "-m", "comment", "--comment", "traffic from SDN", "-j", "ACCEPT"}},
 		{"filter", "INPUT", []string{"-i", "docker0", "-m", "comment", "--comment", "traffic from docker", "-j", "ACCEPT"}},
+		{"filter", "FORWARD", []string{"-s", n.clusterNetworkCIDR, "-m", "conntrack", "--ctstate", "INVALID", "-j", "DROP"}},
```

Comment:
> Adding the drop rule for the invalid packets seems very limited in scope with minimal risk of breaking something.

We *already* broke something. We set up iptables rules that allow invalid packets out of the cluster.

> It seems risky to drop the FORWARD -s clusterNetworkCIDR rule without an understanding of why it was there in the first place

It seems risky to *have* rules that we don't understand why they're there in the first place... the other rules all have comments explaining themselves. We should figure out what this rule is supposed to be doing, and make sure it does that and only that.